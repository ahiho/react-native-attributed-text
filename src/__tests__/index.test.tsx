import React from 'react';
import renderer from 'react-test-renderer';
import AttributedText from '..';

describe('AttributedText', () => {
  describe('Rendering', () => {
    it('should match to snapshot', () => {
      const component = renderer
        .create(<AttributedText>Welcome to Ahiho!</AttributedText>)
        .toJSON();
      expect(component).toMatchSnapshot();
    });
  });
  describe('Interaction', () => {
    it('attributed text onPress should be fire', () => {
      // TODO: add more test case
    });
  });
});
