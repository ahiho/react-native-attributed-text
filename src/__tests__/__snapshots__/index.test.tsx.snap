// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AttributedText Rendering should match to snapshot 1`] = `
<Text>
  Welcome to Ahiho!
</Text>
`;
